samples/react-contoso/package.json

```diff
@@ -9,7 +9,7 @@
     "@fluentui/react": "^8.0.0",
     "@fluentui/react-components": "^9.19.1",
     "@fluentui/react-hooks": "^8.2.0",
-    "@fluentui/react-northstar": "^0.66.4",
+    "@fluentui/react-migration-v0-v9": "^9.1.14",
```

Comment:
IIRC we use the list from northstar in mgt-chat too. Should we be migrating that to use the migration package?